docs/get-started/getting-started-alluxio.md

```diff
@@ -365,3 +365,61 @@ This section will give some links about how to configure, tune Alluxio and some
 - [Running Spark on Alluxio](https://docs.alluxio.io/os/user/stable/en/compute/Spark.html)
 - [Performance Tuning](https://docs.alluxio.io/os/user/stable/en/operation/Performance-Tuning.html )
 - [Alluxio troubleshooting](https://docs.alluxio.io/os/user/stable/en/operation/Troubleshooting.html)
+
+## Alluxio reliability
+### Auto start Alluxio master and workers  
+It's better to add the following crontab jobs to avoid unexpected closes on Alluxio master or workers.  
+Add a crontab job on the Alluxio maser node to automatically start Alluxio maser:  
```

Comment:
It seems odd to me to use crontab job to start Alluxio master every minute to fix the master down issue.

Reply:
I agree, I would rather see us use systemd as well.

Note "master" misspelled above as "maser".